recipes/my-gsm8k.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "MY GSM8K",
```

Comment:
im thinking about the naming of the file, should it be something like gsm8k_1.json or something like that. my-gsm8k may not be representative to the other people on what does 'my' means. The description should include the idea why this is a different one as compared to gsm8k.json.

I think this applies to the otherslike dataset, squad_v2 and so on.

Reply:
Hi @imda-lionelteo , the naming, docstrings and the command line for testing have been updated.